src/services/api.ts

```diff
@@ -12,14 +12,26 @@ export const getInnholdsinformasjon = async (isbn: string): Promise<Innholdsform
     return innholdsinformasjon;
   }
   innholdsinformasjon.title = innholdResponse.title;
-  innholdsinformasjon.image_small = innholdResponse.image_small;
+  innholdsinformasjon.image_path = getImagePath(
+    innholdResponse.image_small,
+    innholdResponse.image_original,
+    innholdResponse.image_large
+  );
   innholdsinformasjon.description_short = splitOnTags(innholdResponse.description_short).map(removeAllTags);
   innholdsinformasjon.description_long = splitOnTags(innholdResponse.description_long).map(removeAllTags);
   innholdsinformasjon.table_of_contents = splitOnTags(innholdResponse.table_of_contents).map(removeAllTags);
 
   return innholdsinformasjon;
 };
 
+function getImagePath(small: string, original: string, large: string) {
```

Comment:
Småpirk, men tenker at å enten rename api.ts fila til utilsFunctions.ts eller flytte disse funksjonene til en egen fil. I en api.ts fil så forventer jeg å kun finne funksjoner som kontakter backend, og evt. støttefunksjoner backendfunksjonene trenger.

Reply:
Jeg (som backender) synes det er ryddig at man spør APIet om innholdsinformasjon fra backenden, og så kan den funksjonen sørge for å gi tilbake et fullverdig innholdsinformasjonsobjekt i stedet for å først hente og så transformere. Men det kan godt være at det ikke er god frontend practice :D

Reply:
Jeg referer ikke til innholdet i denne funksjonen, men de tre andre funksjonene som står i denne fila som ikke kontakter backend

Reply:
Jeg synes det er fint å kunne lese hva som skjer i samme fil, i stedet for å måtte konsultere en annen fil for transformasjonen. Men det er sikkert en smaksak, og igjen - sikkert ikke best practice i frontend?